cclib/method/ddec.py

```diff
@@ -33,18 +33,28 @@ class DDEC6(Method):
     required_attrs = ("homos", "mocoeffs", "nbasis", "gbasis")
 
     def __init__(
-        self, data, volume, proatom_path=None, progress=None, loglevel=logging.INFO, logname="Log"
+        self,
+        data,
+        volume,
+        proatom_path=None,
+        progress=None,
+        convergence_level=1e-10,
+        loglevel=logging.INFO,
+        logname="Log",
     ):
-        # Inputs are:
-        # data -- ccData object that describe target molecule.
-        # volume -- Volume object that describe target Cartesian grid.
-        # proatom_path -- path to proatom densities
-        #      (directory containing atoms.h5 in horton or c2_001_001_000_400_075.txt in chargemol)
+        """ Initialize DDEC6 object.
+            Inputs are:
+                data -- ccData object that describe target molecule.
+                volume -- Volume object that describe target Cartesian grid.
+                proatom_path -- path to proatom densities
+                (directory containing atoms.h5 in horton or c2_001_001_000_400_075.txt in chargemol)
```

Comment:
Perhaps separate this out into a separate paragraph and and explain more. It would be wonderful if you could even summarize what proatom densities are and what role they play in DDEC6. Super great if you can perhaps summarize how the the DDEC6 method works, at a high level. Don't assume everyone looking at the code has read the paper and/or known the ins and outs of all methods out there. However, I think you can assume basic understanding of quantum chemistry and population analysis.

Reply:
I've added some more information below this comment which provides some general idea of how the proatom densities are used and DDEC6 charges are calculated. Please let me know if I missed any important points. Thanks.